broker/lib/controllers/ServiceBrokerApiController.js

```diff
@@ -169,6 +392,13 @@ class ServiceBrokerApiController extends FabrikBaseController {
   }
 
   putBinding(req, res) {
+    if (config.enableServiceFabrikV2) {
+      return this.putBindingV2(req, res);
```

Comment:
Seems we are duplicating a lot of code in putBindingV2 and putBindingV1 methods. Can we combine together and take config driven flow inside a common method. Basically we should avoid code duplication.

Reply:
As discussed. Intention behind doing this was to make it easier to remove 1.0 code entirely.